invenio_records_resources/serializers/xml.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Invenio Resources basic XML serialization module."""
+
+import xmltodict
+from flask_resources.serializers import SerializerMixin
+
+from .json import RecordJSONSerializer
+
+
+class RecordXMLSerializer(SerializerMixin):
+    """Record JSON serializer implementation."""
+
+    def serialize_object(self, obj, response_ctx=None, *args, **kwargs):
+        """Dump the object into an XML string."""
+        if obj:  # e.g. delete op has no return body
+            json_serializer = RecordJSONSerializer()
+            processed_record = json_serializer._process_record(obj)
+            return xmltodict.unparse({"root": processed_record})
+        else:
+            return ""
+
+    def serialize_object_list(
+        self, obj_list, response_ctx=None, *args, **kwargs
+    ):
+        """Dump the object list into an XML string.
```

Comment:
Minor: Don't we want a full example? I think the idea is to have a fully functional one. Not just for demo. @lnielsen can confirm what was the plan :)

Reply:
Definitely would like to hear @lnielsen take because the XML serializer isn't really real: it can be used for Datacite XML or for DublinCore XML and those need to control top-level xml which would be better served by a DataciteXMLSerializer or DublinCoreXMLSerialzer that manipulates those.

Reply:
Hmm I see your point. However, the JSONSerializer is not "real" either. Is plain JSON, is not DataCiteJSON (that comes from RDM-Records datamodel... same goes for XML).

Reply:
Let's shelve this or discuss it at the standup. (This issue, is perhaps getting at the core of it: https://github.com/inveniosoftware/invenio-records-resources/issues/9 better?)